src/main/java/org/jetbrains/bsp/bazel/server/ScalaBspServer.java

```diff
@@ -63,7 +55,8 @@ public ScalaBspServer(
         targetsResolver.getTargetsOptions(targetsUnion, "scalacopts");
     ActionGraphParser actionGraphParser =
         actionGraphResolver.parseActionGraph(
-            MnemonicsUtils.getMnemonics(targetsUnion, Lists.newArrayList(scalac, javac)));
+            MnemonicsUtils.getMnemonics(
+                targetsUnion, Lists.newArrayList(Constants.SCALAC, Constants.JAVAC)));
```

Comment:
is this array modified? if not then it would be better to use `ImmutableList.of(...)`